documentation/source/overview/studies.rst

```diff
@@ -3,6 +3,24 @@
 Studies using SCT
 #################
 
+Summary graphs
+==============
+
+The graphs below summarize the pathologies, tools, species, and count per year of SCT citations:
+
+.. raw:: html
+
+       <iframe width="600" height="371" seamless frameborder="0" scrolling="no" src="https://docs.google.com/spreadsheets/d/e/2PACX-1vSwyEvoiTOMflrJveD277xWYSb_1QSwkpxWsZoMSucgHBS7BHcgfvzGG21--1bLRFO_DIV4EhL9lBl2/pubchart?oid=1220039972&amp;format=interactive"></iframe>
```

Comment:
I've added these snippets in the most straightforward way possible: The HTML formatting here is exactly what's provided by the Google Sheets UI for publishing. 

Still, I think it's worth discussing whether we're happy with the way this looks, or whether we want to try and make it look a different way via some HTML tweaks?